lib/explorers/api-parameters.explorer.ts

```diff
@@ -83,3 +87,25 @@ function removeBodyMetadataIfExplicitExists(
   }
   return properties;
 }
+
+function removeParamMetadataIfExplicitExists(
+    properties: ParamWithTypeMetadata[],
+    explicitParams: any[]
+) {
+
+  const reducer = (a, p) => {
+    if (['query', 'path'].includes(p.in)) {
+      a.push(p.name);
+    }
+    return a;
+  }
```

Comment:
I guess we can rename these parameters

- `a` -> `reflectedParams`
- `p` -> `paramWithTypeMetadata`

Reply:
Thank you for the review. I updated the variable names